nixos/modules/services/networking/kea.nix

```diff
@@ -0,0 +1,363 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.kea;
+
+  format = pkgs.formats.json {};
+
+  ctrlAgentConfig = format.generate "kea-ctrl-agent.conf" {
+    Control-agent = cfg.ctrl-agent.settings;
+  };
+  dhcp4Config = format.generate "kea-dhcp4.conf" {
+    Dhcp4 = cfg.dhcp4.settings;
+  };
+  dhcp6Config = format.generate "kea-dhcp4.conf" {
```

Comment:
Might be confusing to call both these `kea-dhcp4.conf`.

Reply:
Is might copy/paste really that bad? :open_mouth: